tests/test_algorithm.py

```diff
@@ -1223,6 +1225,416 @@ def test_position_weights(self):
             assert_equal(daily_stats.iloc[i]['position_weights'], expected)
 
 
+class TestPortfolio(WithDataPortal, WithSimParams, ZiplineTestCase):
+    START_DATE = pd.Timestamp('2014-01-06', tz='UTC')
+    END_DATE = pd.Timestamp('2016-11-09', tz='UTC')
+    SIM_PARAMS_START = pd.Timestamp('2014-01-10', tz='UTC')
```

Comment:
> Do we really need to run a two year backtest to exercise all the functionality here?

Yes, in order to make sure we cover the full 504 day window length that will be standard for expected shortfall.